sdk/translation/azure-ai-translation-text/src/test/java/com/azure/ai/translation/text/TextTranslationClientBase.java

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.translation.text;
+
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.test.TestBase;
+import com.azure.core.test.TestMode;
+import com.azure.core.util.Configuration;
+
+import java.net.*;
+import java.io.*;
```

Comment:
nit: no `*` imports, if you're using an IDE such as IntelliJ there is a configuration where it folds many imports from the same package into a `*` import, you'll want to change this to a high value to prevent it from doing this folding.

<img width="377" alt="image" src="https://user-images.githubusercontent.com/48699787/227962333-b28d2dca-2ab6-4679-97b4-055b8dd76b61.png">

Reply:
Fixed